p2p/raintree/logging.go

```diff
@@ -0,0 +1,31 @@
+package raintree
+
+import (
+	libp2pNetwork "github.com/libp2p/go-libp2p/core/network"
+
+	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/p2p/utils"
+)
+
+// logStream logs the incoming stream and its scope stats
+func (rtr *rainTreeRouter) logStream(stream libp2pNetwork.Stream) {
+	rtr.logStreamScopeStats(stream)
+
+	remotePeer, err := utils.PeerFromLibp2pStream(stream)
+	if err != nil {
+		rtr.logger.Debug().Err(err).Msg("getting remote remotePeer")
```

Comment:
Why did you go with `Debug` instead of `Error` here?

Ditto below

Reply:
(see below)

In this case, my thinking was that this error only happens in the context of this logging helper function which is not a critical function. I didn't imagine that this logging helper producing an error would be useful to the end user (i.e. not actionable nor a useful signal).